java/src/com/ibm/streamsx/topology/generator/spl/OperatorGenerator.java

```diff
@@ -93,8 +93,12 @@ private void parallelAnnotation(JSONObject op, StringBuilder sb) {
         if (parallel != null && parallel) {
             sb.append("@parallel(width=");
             Object width = op.get("width");
-            if (width instanceof Integer)
+            if (width instanceof Integer) {
                 sb.append(Integer.toString((int) width));
+        	}
```

Comment:
Why is it necessary / what's the benefit of adding code to also support a Long parallel width?  Integer is certainly large enough.